chainforge/react-server/src/NodeLabelComponent.js

```diff
@@ -58,6 +60,30 @@ export default function NodeLabel({ title, nodeId, icon, onEdit, onSave, editabl
         }
     }, [handleRunClick, runButtonTooltip]);
 
+    useEffect(() => {}, [stopButton])
+
+    // useEffect(() => {
+    //     if (stopButton === 'green') {
+    //         setDisplayStopButton(
+    //             <Tooltip label={"Cancel query"} withArrow arrowSize={6} arrowRadius={2} zIndex={1001} withinPortal={true} >
+    //             <button className="AmitSahoo45-button-3 nodrag" onClick={() => handleStopClick(nodeId)}>&#9654;</button>
+    //             </Tooltip>
+    //         );
+    //     } else if (stopButton === 'red') {
+    //         setDisplayStopButton(
+    //             <Tooltip label={"Cancelling query..."} withArrow arrowSize={6} arrowRadius={2} zIndex={1001} withinPortal={true} >
+    //             <button className="close-button nodrag" >&#9208;</button>
+    //             </Tooltip>
+    //         );
+    //     } else if (runButton === 'none') {
+    //         setDisplayStopButton(<button className="AmitSahoo45-button-3 nodrag" onClick={handleRunClick} onPointerEnter={handleRunHover}>&#9654;</button>);
+    //     } else {
+    //         setDisplayStopButton(runButton)
+    //     }
+    // }, [stopButton])
+
+    useEffect(() => {}, [displayStopButton])
```

Comment:
I'm not sure why this line or the above one are necessary. Can you remove these lines 63-85? 

Make sure all code is necessary before you commit it.